mapping_form.html

```diff
@@ -0,0 +1,150 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <title>MANYC Airtable-Gateway Mapping Form</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <style>
+    </style>
+</head>
+
+<body>
+
+    <div class="container">
+        <h1>Welcome to MANYC Airtable-Gateway</h1>
+        <h5>Please enter in your group's Airtable Columns that can correlate with our columns show below...</h5>
+        <form class="form-horizontal" action="api/setMapping" method="POST">
```

Comment:
If `api/setMapping` is an endpoint mounted on the root, `/api/setMapping` feels better to me here. Otherwise I think depending on where the HTML is hosted the request endpoint will be relative to the HTML file location.

Reply:
Agreed. Fixed 👍